web_infrastructure/nginx_status_facts.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2016, René Moser <mail@renemoser.net>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible. If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: nginx_status_facts
+short_description: Retrieve nginx status facts.
+description:
+  - Gathers facts from nginx from an URL having C(stub_status) enabled.
+version_added: "2.3"
+author: "René Moser (@resmo)"
+options:
+  url:
+    description:
+      - URL of the nginx status.
+    required: true
+requirements:
+  - "nginx > 1.3.14"
```

Comment:
However I would of course be happy if you could contribute a patch for earlier versions